pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -0,0 +1,199 @@
+{ lib
+, buildGoModule
+, cryptsetup
+, makeWrapper
+, util-linux
+, which
+, bash
+, coreutils
+, go
+, gpgme
+, libuuid
+  # This is for nvidia-container-cli
+, nvidia-docker
+, openssl
+, squashfsTools
+  # Configurations
+, pname
+, version
+, src
+, projectName # "apptainer" or "singularity"
+  # These cannot be correctly override with `overrideAttrs`
+, vendorSha256 ? null
```

Comment:
This can be done via https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L17= which is a bit none standard.

Reply:
It currently doesn't work, at least when `{ vendorSha256 = null; }` is specified. See #86349.

The injection here is a workaround of that issue.